content/en/synthetics/guide/browser-tests-passkeys.md

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
Editorial suggestion:
Let's keep Passkeys with a capitalized "P"
This is the way Apple talks about it